command/agent/agent.go

```diff
@@ -453,7 +453,10 @@ func (a *Agent) consulConfig() (*consul.Config, error) {
 
 	// Copy the TLS configuration
 	base.VerifyIncoming = a.config.VerifyIncoming || a.config.VerifyIncomingRPC
-	base.VerifyOutgoing = a.config.VerifyOutgoing
+	if a.config.CAPath != "" || a.config.CAFile != "" {
+		base.VerifyOutgoing = true
+	}
+	base.ForceVerifyOutgoing = a.config.VerifyOutgoing
```

Comment:
Instead of remapping `VerifyOutgoing` to `ForceVerifyOutgoing` how about keeping the sense of `VerifyOutgoing` the same and adding something like `UseTLS`?